src/tribol/mesh/CouplingScheme.hpp

```diff
@@ -15,6 +15,18 @@
 
 namespace tribol
 {
+// Struct to hold on-rank coupling scheme face-pair reporting data
+// generated from computational geometry issues
+struct PairReportingData
+{
+public:
+   PairReportingData() {};
+   ~PairReportingData() {};
```

Comment:
Suggestion (minor): This empty constructor and destructor should be removed. 
The destructor's existence removes the default move constructor.

Reply:
Removed. There are a few other enums used for coupling scheme reporting/printing that have empty constructors and destructors. Mind if I remove those too (just those used by the coupling scheme to keep these mods under control)?

Reply:
go for it!